app/lib/main_dev.dart

```diff
@@ -21,6 +21,7 @@ Future main() async {
 
   await Firebase.initializeApp(
     options: DefaultFirebaseOptions.currentPlatform,
+    name: 'ira',
```

Comment:
Why is this required?

Reply:
It was required on my side. There was some issue "Duplicate Firebase APP"

Reply:
Remove this field if this doesn't apply to all contributors.

Reply:
Removed